src/lib/relay/routes.tsx

```diff
@@ -54,15 +54,20 @@ class Gene extends Relay.Route {
 
 class WorksForYou extends Relay.Route {
   static queries = {
-    me: (component, params) => Relay.QL`
+    viewer: (component, params) => Relay.QL`
       query {
-        me {
-          ${component.getFragment("me", params)}
+        viewer {
+          ${(component.getFragment("viewer", params))}
```

Comment:
Hmm, even if just for experimentation purposes and have a better understanding, I’d really like to pair on this with you for a little bit.